### PR TITLE
Adding functioning robot

### PR DESCRIPTION
**Pull request by ThePrinterFRC1799**
added controls and added efficiency to the control chooser.

### CODE REVIEW

**Review by theINKJETprinter — DISMISSED** [review]
Do changes that i requeste

**Review by Fooot-Code — APPROVED** [review]
Hope works please

File: src/main/java/frc/robot/commands/ClimbingDown.java [hunk 1]
```diff
@@ -5,33 +5,43 @@
...
   }
 
   @Override
-  public void initialize() {
-    if (!climbArm.switchLeft.isOk()||!climbArm.switchRight.isOk()) {
-      CommandScheduler.getInstance().cancel(this);
-
+  public void initialize(){
+    if (!m_climbArm.switchLeft.isOk()||!m_climbArm.switchRight.isOk()){
+        CommandScheduler.getInstance().cancel(this);
```
Comment (scottpantall) [inline src/main/java/frc/robot/commands/ClimbingDown.java:32]: Maybe we add something to to SmartDashboard here like "Elevator switches are borked! OHNO!"?

**Review by scottpantall — APPROVED** [review]
Seems good!

**Pull request closed without merging**